eslint.config.mjs

```diff
@@ -0,0 +1,125 @@
+import { fixupPluginRules } from '@eslint/compat';
```

Comment:
this is the new config file. ironically i had to disable lint rules because some of the plugins have not been updated yet. eslint ships a "compat" library for dealing with this, as seen used here